Sources/SwiftKuery/ConnectionPoolConnection.swift

```diff
@@ -16,12 +16,27 @@
 
 // MARK: ConnectionPoolConnection
 
-/// A wrapper around actual `Connection`. Allows seemless release of the connection.
+/**
+ This class uses a `Connection` instance and a `ConnectionPool` instance to implements a wrapper for the `Connection` class.
+ It implements the functions of the `Connection` protocol, in addition to the `closeConnection()` function
+ for releasing a `Connection` instance from the `ConnectionPool`.
+ It is usually initialised by the `ConnectionPool` instance.
```

Comment:
If we don't expect users to need to call this directly it would be helpful to tell users this. We have the following wording for Kitura "We expect most of our users to require higher level concept such as routing, templates and middleware, these are not provided in Kitura-net, if you want to use those facilities you should be coding at the Kitura level, for this please see the Kitura project." perhaps we could add something similar here...